models/user.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require('mongoose');
+const isEmail = require('validator/lib/isEmail');
+const isStrongPassword = require('validator/lib/isStrongPassword');
+// const isLength = require('validator/lib/isLength');
```

Comment:
Why is it here?? If you don't need this remove this, if need uncomment. In other case, you have git to restore git history